examples/core/instancing/app.js

```diff
@@ -59,7 +59,7 @@ const animationLoop = new AnimationLoop({
 
     cube.updateModuleSettings({
       pickingSelectedColor: pickInfo && pickInfo.color,
-      pickingValid: pickInfo !== null
+      pickingSelectedColorValid: pickInfo !== null
```

Comment:
I somewhat strongly feel that the `getUniforms` method should do this `null` check and set the uniforms accordingly, and save the application the trouble. The application would just set `pickingSelectedColor` to `null`.

It's kind of the purpose of `getUniforms` to help the app do the mapping from JavaScript friendly names to uniform names.

Reply:
In case of `autoHighlighting`, we may have non-null `pickInfo` (i.e. non numm pickingSelectedColor), but we should set `pickingSelectedColorValid` to false if highlighting is disabled, this can still be overloaded in JS `getUniforms`, but for this PR given I moving this module from luma.gl would like to keep the change minimal, to avoid any regressions in deck.gl. I will make a note in under API audits.